### PR TITLE
px4-sim formula: add gstreamer dependencies

### PR DESCRIPTION
**Pull request by TSC21**
Pretends to handle the dependency of `sitl_gazebo` on *gstreamer* for MacOS.

### CODE REVIEW

**Comment by TSC21** [comment]
@LorenzMeier seems to me that the formulas need some review in terms of the dependencies. If we take a look at Travis, there seems to be different problems for different OS's and XCode versions.

**Comment by TSC21** [comment]
We probably need to create a specific *gstreamer* formula where we can specifically install *gstreamer1.0*.

File: Formula/px4-sim.rb [hunk 1]
```diff
@@ -9,6 +9,12 @@ class Px4Sim < Formula
   depends_on "exiftool"
   depends_on "glog"
   depends_on "graphviz"
+  depends_on "gst-libav"
+  depends_on "gst-plugins-bad"
+  depends_on "gst-plugins-base"
+  depends_on "gst-plugins-good"
+  depends_on "gst-plugins-ugly"
```
Comment (julianoes) [inline Formula/px4-sim.rb]: I'm pretty sure `base` and `good` are enough.

File: .travis.yml [hunk 2]
```diff
@@ -4,8 +4,13 @@ rvm: system
 language: ruby
 matrix:
   include:
-    - env: OSX=10.13
-      osx_image: xcode10.1
+    - osx_image: xcode9.2
+    - osx_image: xcode9.4
+    - osx_image: xcode10
```
Comment (julianoes) [inline .travis.yml]: What about `xcode10` and newer?

**Pull request merged**